### DISCUSSION TITLE
Array saved as `objects` in `SelectionResult` appeared as `None` in `Production`

### DISCUSSION THREAD

**Discussion by gsaha009** [columnflow/columnflow#429] [General]
Hi CF Experts,

Perhaps a basic problem --->

In the `Selection` step, I am saving a new column `GenTau` as:
```py
SelectionResult(
  steps=bla,
  objects={
    "GenPart" : {
      "GenTau" : indices,
    }
  }
)
```
I have added `GenTau.*` in the `keep_columns` dict in the `config`. Then I have tried to access the column from a `producer` module, and I noticed that all the values of that column are `None`. I added `GenTau.*` in the corresponding `uses` dict as well. Surprisingly, I am able to see the new column using `cf_inspect` on `cf.MergeReducedEvents` `parquet` file, and everything looks fine there. 

I think I am missing something here. Please let me know if you guys have any idea.

Thank you.

**Comment by gsaha009** [comment]
Hi again,

A small finding : 
I just noticed that a few entries were `None` in the first place where I was saving `GenTau` inside the `objects`. I did not know that it would make everything `None`. However, I replaced those `None` values using `ak.where`, still it is all `None` in the `production`. Perhaps, it is for the type!!

**Comment by pkausw** [comment]
Hi @gsaha009 ,

that's strange - including the new columns in the `keep_columns` config section and also specifying them in the `uses` attribute of your producer should have worked. Can you point us to a working minimal example so we can have a look?

Additionally, can you try loading specific columns for your `GenTau` collection to see if it's a problem with matching the columns to load? So something like `GenTau.pt` or something similar

**Comment by pkausw** [comment]
Hi @gsaha009 , there hasn't been any activity in this thread for quite a while. Is this still a problem?

**Comment by gsaha009** [comment]
@pkausw 
This has been resolved. Thanks for the help. closing the discussion.
